apps/src/craft/designer/craft.js

```diff
@@ -703,6 +709,27 @@ Craft.runButtonClick = function() {
     return;
   }
 
+  let store = getStore();
```

Comment:
nit / might be more trouble than it's worth: do we have an existing place to move this code into a shared/util function so we don't have to duplicate it between the different minecraft tutorials?